test/jaxp/javax/xml/jaxp/unittest/validation/SchemaTest.java

```diff
@@ -31,21 +31,94 @@
 import javax.xml.transform.stream.StreamSource;
 import javax.xml.validation.SchemaFactory;
 import org.testng.Assert;
-import org.testng.annotations.Listeners;
+import org.testng.annotations.DataProvider;
 import org.testng.annotations.Test;
 import org.xml.sax.ErrorHandler;
 import org.xml.sax.SAXException;
 import org.xml.sax.SAXParseException;
 
 /*
  * @test
- * @bug 8149915 8222991
+ * @bug 8149915 8222991 8282280 8144117
  * @library /javax/xml/jaxp/libs /javax/xml/jaxp/unittest
  * @run testng/othervm validation.SchemaTest
- * @summary Test Schema creation
+ * @summary Tests schemas and Schema creation.
  */
-@Listeners({jaxp.library.FilePolicy.class})
 public class SchemaTest {
+    private static final String XSD_8144117 = "<?xml version='1.0'?>\n" +
```

Comment:
If it is not from the upstream Xerces test but our own, can we utilize text blocks here?

Reply:
Our source level is still generally 8. The expectation is also that this patch will be backported.